spec/graphql/mutations/users/update_user_optin_status_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+module Mutations
+  module Users
+RSpec.describe UpdateUserOptinStatus, type: :request do
+  describe '.resolve' do
+    it "updates a user's opt-in status" do
+      @rachel = create(:user, id: 1, name: "Rachel Lew", mod: "4")
+
+      result = PairedBeSchema.execute(get_query).as_json
+      expect(result["data"]["getUser"]["name"]).to eq("Rachel Lew")
+      expect(result["data"]["getUser"]["module"]).to eq("4")
+      expect(result["data"]["getUser"]["program"]).to eq("BE")
+      expect(result["data"]["getUser"]["rockOptIn"]).to eq(false)
+
+      post '/graphql', params: { query: query }
+
+      json = JSON.parse(response.body)
+      data = json['data']
+
+      expect(data['user']['rockOptIn']).to eq(true)
```

Comment:
Could you run the post '/graphql', params: { query: query } one more time and expect the user rockOptIn to be false just to validate that it works both ways? Thanks!

Reply:
Good idea – I'll add that to the test.

Reply:
<img width="727" alt="Screen Shot 2020-05-31 at 5 35 03 PM" src="https://user-images.githubusercontent.com/48839191/83365440-55849c80-a365-11ea-9241-844dd210a9c8.png">

Added the extra condition and test still passing. I also tested this in development to be sure (created a new user and saw that the default was false, used the mutation to switch it to true, and used the mutation again to switch it back to false).

Reply:
Thanks Rachel! Looks great!